<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="openglwidget" name="widget" native="true">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 0, 0)</string>
          </property>
          <widget class="circle" name="Circle" native="true">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>189</y>
             <width>91</width>
             <height>91</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="openfile">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>210</y>
             <width>131</width>
             <height>41</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image:url(:/myImage/image/open.png)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>250</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>18%</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(33, 33, 33)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">/*horizontal ：水平QSlider*/
QSlider::groove:horizontal {
border: 0px solid #bbb;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
 /*槽颜色*/
background: rgb(90,49,255);
 /*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:8px;
 /*下遮住区域高度*/
margin-bottom:8px;
/*width在这里无效，不写即可*/
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
/*槽颜色*/
background: rgb(255,255, 255);
/*外环大小0px就是不显示，默认也是0*/
border: 0px solid #777;
/*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:9px;
 /*下遮住区域高度*/
margin-bottom:9px;
}
 
/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 5px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 2px; 
 /*上遮住区域高度*/
margin-top:6px;
 /*下遮住区域高度*/
margin-bottom:6px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 10px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 5px; 
 /*上遮住区域高度*/
margin-top:4px;
 /*下遮住区域高度*/
margin-bottom:4px;
}
</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,4,0,4,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="file_open">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/myImage/image/flie.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:black;
color:white</string>
        </property>
        <property name="text">
         <string>正常</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QPushButton" name="stop">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/myImage/image/stop.png)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previous">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/myImage/image/former.png)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="play">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/myImage/image/play.png)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>21</width>
            <height>21</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/myImage/image/next.png)</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color:white
</string>
            </property>
            <property name="text">
             <string>速度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="styleSheet">
             <string notr="true">/*horizontal ：水平QSlider*/
QSlider::groove:horizontal {
border: 0px solid #bbb;
}

/*1.滑动过的槽设计参数*/
QSlider::sub-page:horizontal {
 /*槽颜色*/
background: rgb(90,49,255);
 /*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:8px;
 /*下遮住区域高度*/
margin-bottom:8px;
/*width在这里无效，不写即可*/
}

/*2.未滑动过的槽设计参数*/
QSlider::add-page:horizontal {
/*槽颜色*/
background: rgb(255,255, 255);
/*外环大小0px就是不显示，默认也是0*/
border: 0px solid #777;
/*外环区域倒圆角度*/
border-radius: 2px;
 /*上遮住区域高度*/
margin-top:9px;
 /*下遮住区域高度*/
margin-bottom:9px;
}
 
/*3.平时滑动的滑块设计参数*/
QSlider::handle:horizontal {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 3px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 2px; 
 /*上遮住区域高度*/
margin-top:6px;
 /*下遮住区域高度*/
margin-bottom:6px;
}

/*4.手动拉动时显示的滑块设计参数*/
QSlider::handle:horizontal:hover {
/*滑块颜色*/
background: rgb(193,204,208);
/*滑块的宽度*/
width: 3px;
/*滑块外环为1px，再加颜色*/
border: 1px solid rgb(193,204,208);
 /*滑块外环倒圆角度*/
border-radius: 5px; 
 /*上遮住区域高度*/
margin-top:6px;
 /*下遮住区域高度*/
margin-bottom:6px;
}
</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="speed">
            <property name="styleSheet">
             <string notr="true">color:white</string>
            </property>
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fullscreen">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image:url(:/myImage/image/fullscreen.png)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>openglwidget</class>
   <extends>QWidget</extends>
   <header>openglwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>circle</class>
   <extends>QWidget</extends>
   <header location="global">circle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
